avocado/core/job.py

```diff
@@ -371,7 +371,7 @@ def _log_avocado_datadir():
         job_log.info('')
 
     def _log_variants(self, variants):
-        lines = variants.str_variants(summary=1, variants=1)
+        lines = variants.to_str(summary=1, variants=1)
```

Comment:
As explained earlier, it was meant for squashing, only I was not sure you'll like the name and more importantly whether you'll like the overall approach...